src/vodman-app/translations/harbour-vodman-sv.ts

```diff
@@ -70,7 +70,7 @@
     <message id="about-sources-text">
         <location filename="../qml/pages/AboutPage.qml" line="149"/>
         <source>Sources are on Github</source>
-        <translation>Källkoder finns tillgängliga på &lt;a href=&quot;https://github.com/jgressmann/harbour-vodman&quot;&gt;GitHub&lt;/a&gt;.</translation>
+        <translation>Källkoder finns på GitHub</translation>
```

Comment:
Please add the link back
```suggestion
        <translation>Källkoder finns på &lt;a href=&quot;https://github.com/jgressmann/harbour-vodman&quot;&gt;GitHub&lt;/a&gt;.</translation>
```